spec/models/task_spec.rb

```diff
@@ -1,5 +1,23 @@
 require 'rails_helper'
 
-RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'バリデーション確認', type: :model do
+  describe 'validation' do
+    let(:task){ FactoryBot.create(:task, title: 'title1', status: 'todo')}
```

Comment:
`FactoryBot.`という値は設定を追記することで省略が可能です。
```suggestion
    let(:task){ create(:task, title: 'title1', status: :todo )}
```